xbmc/threads/platform/pthreads/ThreadImpl.cpp

```diff
@@ -76,6 +76,8 @@ void CThread::SetThreadInfo()
   m_ThreadOpaque.LwpId = pthread_getthreadid_np();
 #elif defined(TARGET_ANDROID)
   m_ThreadOpaque.LwpId = gettid();
+#elif defined(TARGET_DARWIN_TVOS)
```

Comment:
Is there any reason to not use `pthread_mach_thread_np` for all darwin systems?

Reply:
I do not think so, but I have not tested